website/templates/emails/contributor_added_draft_registration.html.mako

```diff
@@ -8,19 +8,10 @@
     %>
     Hello ${user.fullname},<br>
     <br>
-    ${referrer_name + ' has added you' if referrer_name else 'You have been added'} as a contributor to the draft registration "${node.title}" on OSF: ${node.absolute_url}<br>
-    <br>
-    You will ${'not receive ' if all_global_subscriptions_none else 'be automatically subscribed to '}notification emails for this project. To change your email notification preferences, visit your project or your user settings: ${settings.DOMAIN + "settings/notifications/"}<br>
-    <br>
-    If you are erroneously being associated with "${node.title}," then you may visit the draft registration and remove yourself as a contributor.<br>
-    <br>
-    Sincerely,<br>
-    <br>
-    OSF Robot<br>
-    <br>
-    Want more information? Visit https://osf.io/ to learn about the Open Science Framework, or https://cos.io/ for information about its supporting organization, the Center for Open Science.<br>
-    <br>
-    Questions? Email ${osf_contact_email}<br>
-
+    ${referrer_name} has added you to a new registration draft here:
```

Comment:
My impression is that the email should match the second page of this doc (https://docs.google.com/document/d/1ni_oySLZVKN-JcZ_g3NbQIoiYj9a0RAtlwCZFMpw0hE/edit#heading=h.y5oscyvhod6l) which would indicate there's some messaging missing here.

Reply:
Yeah that must have been expanded  after a started the PR, should be updated now.